manifests/init.pp

```diff
@@ -62,7 +62,8 @@
   Boolean $client,
   Boolean $server,
   Boolean $purge,
-  Optional[String] $gem_source = undef
+  Optional[String] $gem_source = undef,
```

Comment:
here I'd just say `String $gem_source` making it required, it should never be undef or unset and we do set it in `common.yaml` so that outcome works.

Otherwise 👍

Reply:
I'm not sure to understand, the line I've added is the 66. I've modified the upper line to add the `,`

Reply:
You're right, man do I need coffee.